app/model/Benchmarks.scala

```diff
@@ -0,0 +1,47 @@
+package model
+
+import ai.x.play.json.Jsonx
+import play.api.libs.json.Format
+import util.DoubleUtils._
+
+case class Totals(uniques: Long, pageviews: Long, timeSpentOnPage: Double)
+object Totals {
+  implicit val totalsFormat: Format[Totals] = Jsonx.formatCaseClass[Totals]
```

Comment:
How come you aren't just using the inbuilt Play `Format` here?

Reply:
It was already in the project. It's essentially play json still just suked up with some extras: https://github.com/xdotai/play-json-extensions